app/Http/Resources/Collection/CollectionResource.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace App\Http\Resources\Collection;
+
+use App\Http\Resources\Auth\UserResource;
+use Illuminate\Http\Request;
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class CollectionResource extends JsonResource
+{
+    /**
+     * Transform the resource into an array.
+     *
+     * @return array<string, mixed>
+     */
+    public function toArray(Request $request): array
+    {
+        return [
+            'name' => $this->name,
+            'user' => new UserResource($this->user),
```

Comment:
I think `user` property is unnecessary in this position because every user can view their own collections